src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -105,27 +133,103 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
-    <div>
-      <NERButton
-        endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
-        variant="contained"
-        id="unreviewed-cr-actions-dropdown"
-        onClick={handleClick}
-      >
-        Actions
-      </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
-        <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
-          Review
-        </MenuItem>
-        <Divider />
-        <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
-          Delete
-        </MenuItem>
-      </Menu>
-    </div>
-  );
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) return;
+    try {
+      await requestCRReview.mutateAsync({ userIds: reviewerIds });
+    } catch (e) {
+      if (e instanceof Error) {
+        toast.error(e.message);
+      }
+    }
+  };
+
+  const isRequestAllowed = changeRequest.submitter.userId === currentUser.userId;
+
+  const unreviewedActionsDropdown = () => {
+    if (isRequestAllowed && changeRequest.status === ChangeRequestStatus.Open) {
+      return (
+        <div>
+          <NERButton
+            endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
+            variant="contained"
+            id="unreviewed-cr-actions-dropdown"
+            onClick={handleClick}
+          >
+            Actions
+          </NERButton>
+          <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+            <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
+              Review
+            </MenuItem>
+            <Divider />
+            <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
+              Delete
+            </MenuItem>
+          </Menu>
+          <Grid container spacing={2}>
+            <Grid item xs={12} md={8}>
+              <Autocomplete
+                isOptionEqualToValue={(option, value) => option.id === value.id}
+                limitTags={1}
+                disableCloseOnSelect
+                multiple
+                options={users.filter((user) => isLeadership(user.role)).map(userToAutocompleteOption)}
+                getOptionLabel={(option) => option.label}
+                onChange={(_, values) => setReviewerIds(values.map((value) => value.id))}
+                renderOption={(props, option, { selected }) => (
+                  <li {...props}>
+                    <Checkbox
+                      icon={<CheckBoxOutlineBlankIcon />}
+                      checkedIcon={<CheckBoxIcon />}
+                      style={{ marginRight: 8 }}
+                      checked={selected}
+                    />
+                    {option.label}
+                  </li>
+                )}
+                renderTags={() => null}
+                renderInput={(params) => (
+                  <TextField {...params} variant="standard" label="Add Reviewers" placeholder="Select Reviewers" />
```

Comment:
change the placeholder to `Choose a user(s)`, remove the label prop, and make it so that the selected options are shown in this textfield once selected

Reply:
@RChandler234 Good points. 

Although, when we display the selected options in the textfield, the height of textfield gets large as the user selects more:
<img width="946" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/91857946/aa81e09f-1ce2-4e49-972f-a588d2654613">
<img width="950" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/91857946/20e4c5da-d8c7-479c-b4be-75545c7de5c0">

This will make a large blank space when the user clicks on the textfield and selects multiple reviewers. Also, the textfield goes down as the user selects more reviewers. I think leaving the selected options as just check mark will be also great in terms of UX (like how Github pr reviewer feature is set up), but lmk your thoughts. You can try it on your end by just removing line 191 (renderTags={() => null}).

Reply:
Github PR Reviewers actually shows the selected reviewers, but I see your point. Could you just display like '# Reviewers Selected' in the select, just so that the user has some way of knowing they've correctly selected reviewers when the dropdown is closed. I think this will be better addressed in the complete redesign as wel